routes/api_worldometers.js

```diff
@@ -20,6 +21,7 @@ const getAllData = async (key) => {
 
 const fixApostrophe = (country) => {
 	country.country = country.country.replace(/"/g, '\'');
+	delete country.continent;
```

Comment:
is this really needed? 
i think i would like to leave it in but leaving it in could lead to the problem of having "continent" as the equivalent to "country" but at the same time, a country object also has the property "continent"

ready for discussion

Reply:
Idk, what do you think @ebwinters ?

Reply:
We have the `country` field in `/countries` endpoint so I think we have to leave it for consistency.

Reply:
I gonna leave that property